spec/lib/dqt/objects/teacher_spec.rb

```diff
@@ -359,8 +385,8 @@ def response(response)
   describe "#degree_codes" do
     subject(:degree_codes) { qualified_teaching_status.degree_codes }
 
-    it "returns Array" do
-      expect(degree_codes).to eq []
+    it "returns an Array" do
+      expect(degree_codes).to eq ["100403", "100105"]
```

Comment:
That isn't what you're asserting though. I'd just go with `specify { expect(degree_codes).to contain_exactly("100403", "100105")}` and let `specify` generate you a failing message if it fails (rather than relying on what you typed)

Reply:
Done in 8c6078e31a2a1d16a4e1074f3c7f68168eb68acd